browser-extension/src/models/tournesolContainer/TournesolContainer.js

```diff
@@ -0,0 +1,168 @@
+import { TournesolVideoCard } from '../tournesolVideoCard/TournesolVideoCard.js';
+
+export class TournesolContainer {
+  constructor(parent, banner) {
+    this.isExpanded = false;
+    this.parent = parent;
+    this.banner = banner;
+  }
+
+  createHTMLElement() {
+    // Create container
+    let tournesol_container = document.createElement('div');
+    tournesol_container.id = 'tournesol_container';
+
+    // Add inline-block div
+    const inline_div = document.createElement('div');
+    inline_div.setAttribute('class', 'inline_div');
```

Comment:
This div is not inline anymore, let's change the class name and the comment.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1436/commits/32084d1d13b49c909f378edfe57e5069137fc141